### PR TITLE
Run builds on every pull request

### PR DESCRIPTION
**Pull request by hikhvar**

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+  push:
+    branches: ["master"]
+  pull_request:
 jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
     steps:
-    - name: Set up Go 1.13
+    - name: Set up Go 1.15
```
Comment (arnisoph) [inline .github/workflows/go.yml:11]: maybe remove the version at all?

Comment (hikhvar) [reply .github/workflows/go.yml:11]: I like deterministic builds. Pinning the minor version is the best we can do.

Comment (arnisoph) [reply .github/workflows/go.yml:11]: I'm commenting L11 only here. just the name

Comment (hikhvar) [reply .github/workflows/go.yml]: I removed it. Sorry answered on my phone. Didn't saw the complete code.

**Pull request merged**